.github/workflows/third-party-library-changes.yml

```diff
@@ -0,0 +1,63 @@
+name: Third Party Library Changes
+
+on:
+  pull_request:
+    paths:
```

Comment:
Should we also add `branch` qualifiers? Maybe `trunk`, and `6.[7-9]`/`[7-9].[0-9]` to run on the current version and all future ones?